### DISCUSSION TITLE
Copilot - a tiny feature with big impact

### DISCUSSION THREAD

**Discussion by cmp-nct** [community/community#36207] [Copilot Conversations]
I'm on Jetbrains PHPStorm, please add this as soon as you can:

I need a shortcut that completes ONE character of the completion text.
I would like to be able to just spam it or keep it pressed.

I have had a TON of cases where I needed a completion of a part of a line but to accept it I have to add 80% wrong code that later needs to be removed manually again.
So I just want to press a key and hold it or quickly press it and it will write for me until I am done.

Please add that, it's a tiny feature you can probably add in minutes and will make my life with copilot so much more useful.

Currently I often give up on the completion, I write it myself because accepting it is more hassle than good.

**Comment by iamcjwilliams** [comment]
@comp-nect Thank you for your feature suggestion!

**Comment by cmp-nct** [comment]
I developed a plugin that's doing what I described, not as refined as copilots internal handling of the IDE but working surprisingly well.
https://github.com/cmp-nct/Stewardess
It is made to work seamless with copilot, it will not interrupt it's typical workflow.

Should be considered "alpha", it's working on my IDE but might have bugs.
It also could do a better job splitting, more "code oriented" than "word oriented".
